lib/improver/ensemble_copula_coupling/ensemble_copula_coupling_constants.py

```diff
@@ -64,14 +64,14 @@
     "precipitation_rate": Bounds((0, 0.00003), "m s-1"),
     "rainfall_rate": Bounds((0, 0.00003), "m s-1"),
     "rainfall_rate_in_vicinity": Bounds((0, 0.00003), "m s-1"),
-    "relative_humidity": Bounds((0, 2.0), "1"),
+    "relative_humidity": Bounds((0, 1.2), "1"),
```

Comment:
I think there was a reason this upper-bound was so high. Something to do with relative humidity with respect to water, but over ice in Antarctica. I believe this combination leads to much higher values. @fionaRust may know more.

Reply:
See plots on IMPRO-757. It doesn't necessarily mean that we want it this high, as the code will now cope with these situations better.

Reply:
Thanks.
I agree that the code will now cope with values higher than these bounds in a sensible way, so we will leave this as Mark has set it.